Stack_Array1.java

```diff
@@ -0,0 +1,127 @@
+	//Precourse 1: Exercise 1    --  Implementing stack using array
+	// Time Complexity : O(n) because there is traversal from bottom or first index till last in order to add item at last+1 position. 

```

Comment:
No, For insertion, deletion and peek() time complexity is constant. As you have access to index so that's why it is not O(N).